storybook/storybook-react/src/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -15,14 +15,12 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const BreadcrumbStory: Story = {
+export const Default: Story = {
   render: () => (
     <Breadcrumb>
-      <Breadcrumb.Item href="#">home</Breadcrumb.Item>
-      <Breadcrumb.Item href="#">path 1</Breadcrumb.Item>
-      <Breadcrumb.Item href="#">path 2</Breadcrumb.Item>
-      <Breadcrumb.Item href="#">path 3</Breadcrumb.Item>
-      <Breadcrumb.Item href="#">path 5</Breadcrumb.Item>
+      <Breadcrumb.Item href="#">Home</Breadcrumb.Item>
+      <Breadcrumb.Item href="#">Nieuws</Breadcrumb.Item>
+      <Breadcrumb.Item href="#">Kennisgevingen en bekendmakingen</Breadcrumb.Item>
```

Comment:
A page depth of 5 like the Figma examples will help development for small screens, I think breaking the lines instead of a horizontal scroll will cause a lot of vertical space.

Reply:
This was the longest example I could find on our current website, so maybe a depth of 5 isn’t all that realistic.

I don’t think breadcrumb items should overflow the screen. That’s a useful pattern for a menu, but applying it to breadcrumbs would hide the most important items from sight.

Reply:
Agreed, I did find one [example](https://www.gov.uk/plan-shared-parental-leave-pay) where there are 3 long breadcrumbs and when viewing on mobile they hide the second item.